packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/utils/api.js

```diff
@@ -0,0 +1,11 @@
+import { axiosInstance } from '../../../../../../core/utils';
+
+const fetchData = async () => {
+  const {
+    data: { results },
+  } = await axiosInstance.get(`/admin/audit-logs?pageSize=30`);
```

Comment:
Is there any reason for that 30 number? I think until we have pagination we could set 100 as otherwise you lose access to a lot of data